recipe/conda_build_config.yaml

```diff
@@ -61,3 +75,7 @@ cdt_name:  # [linux]
   - cos7   # [linux64]                  # CUDA 11.0
   - cos7   # [linux64]                  # CUDA 11.1
   - cos7   # [linux64]                  # CUDA 11.2
+
+  - cos7   # [linux and ppc64le]        # CUDA 10.2
+  # - cos7   # [linux and ppc64le]        # CUDA 11.0
```

Comment:
Why not add these? If we are going to add a migrator, we should add these two. Otherwise we'll have to have 2 migrators.

Reply:
I left those there as a reminder while I was submitting the PR to `docker-images`. They are now added.